src/Controls/src/Core/Element.cs

```diff
@@ -638,5 +638,7 @@ private protected virtual void OnParentChangingCore(Element oldParent, Element n
 			ParentChanging?.Invoke(this, args);
 			OnParentChanging(args);
 		}
+
+		IReadOnlyList<object> IVisualTreeHelper.GetVisualChildren() => LogicalChildren;		
```

Comment:
@hartez is this still LogicalChildren? I recall a PR somewhere that did something with this property. Not sure if it was additive or now is a bigger change.